### ISSUE TITLE
Error When Running Get-Action1 RawURI 

### ISSUE THREAD

**Issue by Esnapp** [Action1Corp/PSAction1#4]
When running Get-Action1 RawURI  (or presumably any other functions that use Invoke-WebRequest) on a system without internet explorer (or where first time setup of IE hasn't been completed) you get the error:
`Invoke-WebRequest : The response content cannot be parsed because the Internet Explorer engine is not available, or Internet Explorer's first-launch configuration is not complete. Specify the UseBasicParsing parameter and try again. At C:\Program Files\WindowsPowerShell\Modules\PSAction1\1.3.7\PSAction1.psm1:220 char:54 + ... nputObject (Invoke-WebRequest -Uri "$Script:Action1_BaseURI/oauth2/to ... + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ + CategoryInfo : NotImplemented: (:) [Invoke-WebRequest], NotSupportedException + FullyQualifiedErrorId : WebCmdletIEDomNotSupportedException,Microsoft.PowerShell.Commands.InvokeWebRequestComman d`

To resolve this you can add the line `$PSDefaultParameterValues['*:UseBasicParsing'] = $true` to `PSAction1.psm1 ` to set the script to use basic parsing as none of the calls require DOM parsing.

**Comment by GeneMoody-Action1** [comment]
This is specified at a per call level so while not specified across the whole script it is at every place Invoke-WebRequest is used.

**Comment by Esnapp** [comment]
Yep looks like this change was made back in January with this commit https://github.com/Action1Corp/PSAction1/commit/3598319142f0130aa32d389dc942ecf5d263304f . At the time of the issue it was still not using basic parsing. I'll go ahead and mark this as closed, thanks.
